src/main/java/com/datadog/api/v1/client/JavaTimeSerializer.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache-2.0 License.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.datadog.api.v1.client;
```

Comment:
do we need the same file for each package?

Reply:
We don't but we already do this everywhere else. We have plans to extract all of this out into another package in a separate pr